cmd/pbstatuser/main.go

```diff
@@ -90,6 +90,8 @@ func processMessage(ctx context.Context, message *kafka.GenericMessage) {
 		Identity:           ctxval.Identity(ctx),
 	}
 
+	logger.Trace().Msgf("received a message from sources to be processed with source application id %s", s.SourceApplictionID)
```

Comment:
Can we put this higher? Most failures I've seen seemed to be in the fetching of the authentication, so those would not get to this line IMO